models/BackendUser.php

```diff
@@ -33,7 +33,7 @@ class BackendUser extends BackendUserModel implements AuthenticatableContract
      *
      * @var string
      */
-    protected $rememberTokenName = 'persist_code';
-
-
+    public function __construct() {
+        $rememberTokenName = 'persist_code';   
```

Comment:
What's the point of this change in particular? What was wrong with it being a property?

Reply:
@LukeTowers Maybe I'm wrong, but I got an error said that is was declared multiple times. So thought that this would fix this

Reply:
I, too, got an error:

`LukeTowers\Passport\BackendUser and Illuminate\Auth\Authenticatable define the same property ($rememberTokenName) in the composition of LukeTowers\Passport\BackendUser. However, the definition differs and is considered incompatible. Class was composed`

Applying the changes suggested here fixed the error.